src/StackAdmin/Azs.Subscriptions/Module/Azs.Subscriptions/Azs.Subscriptions.psd1

```diff
@@ -56,8 +56,8 @@ Licensed under the MIT License. See License.txt in the project root for license
     # ProcessorArchitecture = ''
 
     # Modules that must be imported into the global environment prior to importing this module
-    RequiredModules      = @(@{ModuleName = 'AzureRM.Profile'; RequiredVersion = '3.4.1'; },
-                             @{ModuleName = 'AzureRM.Resources'; RequiredVersion = '4.4.1'; })
+    RequiredModules   = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.3.4'; },
```

Comment:
Update to 5.5.1 for AzureRm.Profile

Reply:
Is that what we are doing for Tenant module?

Reply:
Resolved